cosmos-sdk-proto/src/prost/cosmos-sdk/cosmos.app.v1alpha1.rs

```diff
@@ -112,7 +112,7 @@ pub struct ModuleConfig {
     /// config is the config object for the module. Module config messages should
     /// define a ModuleDescriptor using the cosmos.app.v1alpha1.is_module extension.
     #[prost(message, optional, tag = "2")]
-    pub config: ::core::option::Option<::tendermint_proto::google::protobuf::Any>,
+    pub config: ::core::option::Option<::pbjson_types::Any>,
```

Comment:
These should stay as they are. Instead of using `pbjson-types`, you can leverage the newly added `pbjson` support in `tendermint-proto` (which solves every downstream crate having to figure out this problem)

Reply:
After updating the extern_paths, these now generate as they did before